src/SAML2/HTTPArtifact.php

```diff
@@ -66,8 +66,8 @@ public function getRedirectURL(Message $message) : string
         if ($destination === null) {
             throw new \Exception('Cannot get redirect URL, no destination set in the message.');
         }
-        /** @psalm-suppress UndefinedClass */
-        return HTTP::addURLparameters($destination, $params);
+        $utilsHttp = new HTTP();
```

Comment:
Should we use the one from the container instead of instantiating a new one?

Reply:
Not sure it's worth the extra interface and code for one instance across the codebase?

Reply:
The thing is, HTTP-Artifact can now only be used together with SSP, not just with the standalone library.
Using the container allows for DI

Reply:
Not sure how to do this in a way that keeps the container interface generic. Maybe solve that separately from this buffix PR.

Reply:
Oh but wait, we moved the SSPContainer to SSP because it didn't belong here..  See SSP@master

Reply:
So this fix is ok for 4.x I presume.

Reply:
Yeah, let's leave it as is.. I don't see a way to properly deal with this without breaking backwards compatibility, so it'll have to wait for 5.x